src/ophys_etl/workflows/db/initialize_db.py

```diff
@@ -0,0 +1,162 @@
+"""Script to create and populate tables"""
+from typing import Dict
+
+import argschema
+from sqlalchemy import create_engine, event
+from sqlmodel import SQLModel, Session
+
+from ophys_etl.workflows.db.db_utils import fk_pragma_on_connect
+from ophys_etl.workflows.db.schemas import Workflow, WorkflowStep, \
+    WellKnownFileType
+
+
+class InitializeDBSchema(argschema.ArgSchema):
+    db_url = argschema.fields.String(
+        required=True,
+        description='db url'
+    )
+
+
+def create_db_and_tables(engine):
+    """Creates db and empty tables"""
+    SQLModel.metadata.create_all(engine)
+
+
+def _create_workflow(session):
+    """Adds Workflow"""
+    workflow = Workflow(name='ophys_processing')
+    session.add(workflow)
+    session.commit()
+    return workflow
+
+
+def _create_workflow_steps(session, workflow: Workflow):
+    """Adds workflow steps"""
+    workflow_steps = {
+        'motion_correction': WorkflowStep(
+            name='motion_correction',
+            workflow_id=workflow.id
+        ),
+        'denoising_finetuning': WorkflowStep(
+            name='denoising_finetuning',
+            workflow_id=workflow.id
+        ),
+        'denoising_inference': WorkflowStep(
+            name='denoising_inference',
+            workflow_id=workflow.id
+        ),
+        'segmentation': WorkflowStep(
+            name='segmentation',
+            workflow_id=workflow.id
+        ),
+        'roi_classification': WorkflowStep(
+            name='roi_classification',
+            workflow_id=workflow.id
+        ),
+        'trace_extraction': WorkflowStep(
+            name='trace_extraction',
+            workflow_id=workflow.id
+        )
+    }
+    for workflow_step in workflow_steps.values():
+        session.add(workflow_step)
+    session.commit()
+    return workflow_steps
+
+
+def _create_motion_correction_well_known_file_types(
+    session,
+    workflow_steps: Dict[str, WorkflowStep]
+):
+    well_known_file_types = [
+        WellKnownFileType(
+            name='OphysMaxIntImage',
```

Comment:
Renamed to `MAX_INTENSITY_PROJECTION_IMAGE`. Also improved by using enum in `well_known_file_types.py` so that we can refer to 1 place for the value instead of using strings throughout the code